lastminute.yml

```diff
@@ -21,6 +22,13 @@
     - role: ldapserver
       tags: ldapserver
 
+- name: configure the squid server
+  hosts: squidserver
```

Comment:
Is this now the wrong groupname?

Reply:
should we also rename that 205 host?  to what?